server/api/shared.py

```diff
@@ -92,6 +92,31 @@ def batch_tallies(contest: Contest) -> BatchTallies:
     }
 
 
+class CombinedBatch(TypedDict):
+    name: str
+    representative_batch: Batch
+    sub_batches: List[Batch]
+
+
+def combined_batch_representative(sub_batches: List[Batch]) -> Batch:
+    assert len(sub_batches) > 0
+    sampled_sub_batches = [sub_batch for sub_batch in sub_batches if sub_batch.draws]
+    return sorted(sampled_sub_batches, key=lambda batch: batch.id)[0]
```

Comment:
Good question, I think when I wrote it I decided that crashing was fine, since there would be no motivation for a support user to try to combine batches without including a sampled batch lol. But being more explicit couldn't hurt. I'll add it